gyp/pylib/gyp/input.py

```diff
@@ -233,7 +233,12 @@ def LoadOneBuildFile(build_file_path, data, aux_data, includes,
     # Open the build file for read ('r') with universal-newlines mode ('U')
     # to make sure platform specific newlines ('\r\n' or '\r') are converted to '\n'
     # which otherwise will fail eval()
-    build_file_contents = open(build_file_path, 'rU').read()
+    if sys.platform == 'zos':
```

Comment:
Did you try to upstream it. Or in other words, will it be a problem for `node` (if we have non ASCII chars, possibly in `config.gypi` via `node-prefix` for example) ?

Or will it be better that `node-gyp` will generate UTF-8-BOM files, and skip patching GYP?

Reply:
@refack I tried looking for what `node-prefix` does. but couldn't find anything. Can you point me to some documentation about it?